api/src/users/api_views.py

```diff
@@ -46,6 +51,21 @@ class ConnectUser(GenericUserExtension):
     def field_func(obj_id):
         return CustomUser.objects.get(id=obj_id).connected_users
 
+    def post(self, request, *args, **kwargs):
+        response: Response = GenericUserExtension.post(self, request, args, kwargs)
```

Comment:
You can remove the type hint for this since post() always returns a Response object.